app/src/androidTest/java/com/example/resoluteapp/LogExercise_IT.java

```diff
@@ -85,4 +84,38 @@ public void clearTextFieldsAfterSuccessfulLog() throws InterruptedException {
         onView(withId(R.id.editTextNumberOfUnits)).check(matches(withText("")));
 
     }
+
+    @Test
+    public void sendExerciseToFriend() throws InterruptedException {
+
+        // Log an exercise
+        onView(withId(R.id.editTextExercise)).perform(typeText("UNIQUE"));
+        onView(withId(R.id.editTextUnits)).perform(typeText("Units"));
+        onView(withId(R.id.editTextNumberOfUnits)).perform(typeText("Amounts"), closeSoftKeyboard());
+        onView(withId(R.id.log_exercise_button)).perform(click());
+        Thread.sleep(2000);
+
+        // Log Out and Relaunch
+        emptySharedPref();
+        activityScenarioRule.getScenario().close();
+        ActivityScenario.launch(MainActivity.class, null);
+
+        // Log In As Friend
+        onView(withId(R.id.login_username_entry)).perform(typeText("Friend"), closeSoftKeyboard());
+        onView(withId(R.id.login_password_entry)).perform(typeText("Password"), closeSoftKeyboard());
+        onView(withId(R.id.login_button)).perform(click());
+
+        // Navigate to Inbox
+        Thread.sleep(2000);
+        onView(withId(R.id.to_inbox_from_home)).perform(click());
+        Thread.sleep(2000);
+
+        onView(withId(R.id.tableLayout)).check(matches(isDisplayed()));
+        onView(allOf(withText("UNIQUE"))).perform(click());
+
```

Comment:
If you send a reply, it deletes the item from the inbox. To make the test continuous you can just send a reply. 
One line: 
onView(withText("Send")).perform(click());